app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -596,13 +596,18 @@ private void updateThumbnailUrl() {
 
         changeThumbnailUrl(newThumbnailUrl);
     }
-
     private void deleteItem(final PlaylistStreamEntry item) {
         if (itemListAdapter == null) {
             return;
         }
+        new AlertDialog.Builder(getContext())
+                .setTitle(R.string.confirm_video_deletion_from_local_playlists)
+                .setNegativeButton(R.string.cancel, null)
+                .setPositiveButton(R.string.ok,
+                        (dialog, which) -> itemListAdapter.removeItem(item))
+                .create()
+                .show();
 
-        itemListAdapter.removeItem(item);
         if (playlistManager.getPlaylistThumbnail(playlistId)
```

Comment:
All of the actions below should also be done only if the user really wants the item to be removed